src/nxdoc/nuxeo-server/installation/nuxeo-cluster-architecture/nuxeo-architecture-components/asynchronous-processing-architecture.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Asynchronous Processing
+description:
+review:
+    comment: ''
+    date: '2021-03-10'
+    status: ok
+toc: true
+tree_item_index: 700
+---
+
+Asynchronous processing relies on the **WorkManager**, **Nuxeo Stream**, **KeyValue Store** and the **PubSub Service**.
```

Comment:
Different Nuxeo services rely on a messaging system, this is the case for the WorkManager, Nuxeo Stream, the KeyValue Store and the PubSub services.